docs/plugins.rst

```diff
@@ -28,6 +28,9 @@ The python modules in the ``plugins`` folder get imported,
 and **hooks**, **operators**, **sensors**, **macros** and web **views**
 get integrated to Airflow's main collections and become available for use.
 
+To troubleshoot issue with plugins, you can use ``airflow plugins`` command.
+This command dumps information about loaded plugins.
+
```

Comment:
Can we add an example here?

Reply:
I can add, but I'm not sure it will be useful.  This tool displays a lot of information  When I had one plug-in installed, 56 lines of text were displayed.  We don't have an example in the documentation for the airflow info command either. This is a similar type of command. It displays a lot of information, but it doesn't matter how it looks. On the other hand, we have a description of the `airflow dag show`  command, where the example gives a better understanding of the command.

Reply:
Ignore it for now then